src/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { PrevArrow } from '../icons/buttons-icons/prev-arrow';
+import { NextArrow } from '../icons/buttons-icons/next-arrow';
+import { theme } from '../layout/theme';
+import { PaginationDropdown } from '../typography/typography';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+
+interface PaginationProps {
+  onPreviousClick: () => any;
+  onNextClick: () => any;
+  onDropdownSelect: (selectedPage: number) => any;
+  amountOfPages: number;
+  currentPage: number;
+}
+
+const PaginationContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const Dropdown = styled(PaginationDropdown)`
+  background: none;
+  border: none;
+  color: ${theme.palette.text.link.default};
+  cursor: pointer;
+`;
+
+const StyledButton = styled.button<{ inactive: boolean }>`
+  background: none;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+
+  ${(props) =>
+    props.inactive &&
+    css`
+      cursor: default;
+      opacity: 50%;
+    `}
+`;
+
+export const Pagination: React.FC<PaginationProps> = ({
```

Comment:
Using `React.FC<...>` is outdated and shouldn't be used anymore. It better to use the props interface directly and `JSX.Element` as return type.

```ts
export const Pagination = (props: PaginationProps): JSX.Element => {}
```

This blog post goes into detail about this topic: https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/